bare-metal/elastic-metal/how-to/send-metrics-logs-to-cockpit.mdx

```diff
@@ -0,0 +1,53 @@
+---
+meta:
+  title: How to send metrics & logs from bare metal servers to Cockpit
+  description: This page explains how to send metrics & logs from bare metal servers to Cockpit
+content:
+  h1: How to send metrics & logs from bare metal servers to Cockpit
+  paragraph: This page explains how to send metrics & logs from bare metal servers to Cockpit
+tags: metrics grafana cockpit elastic-metal
+dates:
+  validation: 2024-02-13
+  posted: 2024-02-13
+categories:
+  - bare-metal
+---
+
+Monitoring your dedicated servers with [Scaleway Cockpit](/observability/cockpit/quickstart/) offers detailed information about performance and health, ensuring optimal functionality and proactive issue resolution.
+Cockpit provides a user-friendly interface for real-time monitoring of CPU, memory, disk usage, and network activity, enhancing the reliability and performance of bare metal servers.
+
```

Comment:
I think Maxime mentioned adding a message box to warn about additional costs that might apply. Maybe copy/paste the one this page https://www.scaleway.com/en/docs/observability/cockpit/how-to/send-metrics-logs-to-cockpit/